modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -152,23 +289,27 @@ const CustomMenu = () => {
 };
 ```
 
-#### Layout and Column
+🤖 The codemod will rename `Menu`, `MenuProps`, `MenuState`, `MenuItem` and `MenuItemProps` to their
+deprecated names.
 
-We are [soft deprecating](#soft-deprecation) `Layout` and `Column`. It has been renamed to
-`DeprecatedLayout` and `DeprecatedColumn`. Similarly, `LayoutProps` has been renamed to
-`DeprecatedLayoutProps`, `Column` to `DeprecatedColumn` and `ColumnProps` to
-`DeprecatedColumnProps`. You may continue to use this component, but note that we plan to
-[hard-deprecate](#hard-deprecation) it in Canvas Kit v9.
+You may continue to use this component, but please note that we plan to
+[hard-deprecate](#hard-deprecation) it in v9. Consider using the [Menu](/components/popups/menu/) in
+the Main package instead.
```

Comment:
Note that `/components/popups/menu/` will link to docs for the **Preview** Menu when the v8 Upgrade Guide is viewed on the Canvas Site (unless we update the site to import the Main Menu MDX instead of the Preview Menu MDX). This link will work fine when viewing the v8 Upgrade Guide within Storybook.

Reply:
We'll update the site to import the Main Menu MDX instead of the Preview Menu MDX.